text/0000-deprecation.md

```diff
@@ -0,0 +1,243 @@
+- Feature Name: Target Version
+- Start Date: 2015-06-03
+- RFC PR: 
+- Rust Issue: 
+
+# Summary
+
+This RFC proposes a small number of extensions to improve the user
+experience around different rust versions, while at the same time
+giving Rust developers more freedom to make changes without breaking
+anyones code.
+
+Namely the following items:
+
+1. Add a `--target=`*<version string> command line argument to rustc.
+This will be used for deprecation checking and for selecting code paths
+in the compiler.
```

Comment:
rustc already has a `--target` flag, used for a completely separate purpose (selecting target triples).